doc/nrf/ug_thread.rst

```diff
@@ -58,10 +58,10 @@ Required modules
 Thread requires the following Zephyr's modules to properly operate in |NCS|:
 
 * :ref:`zephyr:ieee802154_interface` radio driver - This library is automatically enabled when working with OpenThread on Nordic Semiconductor's Development Kits.
+  Its advanced features are implemented through :ref:`nrfxlib:nrf_802154_sl` in `nrfxlib`_.
```

Comment:
```suggestion
  Its advanced features are implemented through the :ref:`nrfxlib:nrf_802154_sl` in `nrfxlib`_.
```
It resolves to "... Service Layer", correct?

Reply:
Correct.